tools/src/test/resources/1.4/valid-vulnerability-1.4.xml

```diff
@@ -116,6 +116,12 @@
                     </versions>
                 </target>
             </affects>
+            <properties>
+                <property name="Foo">Bar</property>
```

Comment:
your currently suggested XSD does not specify any children inside the  `properties` element

Reply:
IIUC the XSD is saying that adding properties of type "bom:propertiesType" is optional.

Maybe I'm misunderstanding, but in the same way that [this `valid-properties-1.4.xml` mirrors that of its JSON counterpart](https://github.com/CycloneDX/specification/blob/c3da323e637e4fbe770126ce3359faa9770f8743/tools/src/test/resources/1.4/valid-properties-1.4.xml#L4-L9), I thought it was important to bring this to `properties` snippet to this vulnerability _test_ file in order to verify that this XML complies with the updated schema. Without the corresponding XML schema change, [we know from this PR that an error that the XML will not comply and thus error](https://github.com/CycloneDX/cyclonedx-go/pull/39), which gave rise to the issue that this PR is designed to close.

Does this make sense..? I could very well be wrong as I don't know how to run tests / develop locally in this repo (didn't see it in the README), but if you wouldn't mind sharing with me how to do that, I'd be happy to do that and post test results. (I think it'd be a good addition to the README anyhow which I don't mind doing.)

Reply:
got it. 
/closed